unfoldingWord/translationCore
# Removing all text within footnotes (and other filtered markers)

#### This pull request addresses:
This PR adjusts the regex to capture all the text inclosing a filtered marker within a text.


#### How to test this pull request:
run ```npm update usfm-js```
Open a php project.
Open the tW tool. 
Look at the first check ```amen, truly``` ensure there is no text such as ```Some version keep Amen.``` in the scripture pane (This used to be here as text given from the footnote)
Try finding other examples of markers that have other text enclosed.
Related story is #1941


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3078)
